src/applications/vaos/tests/covid-19-vaccine/components/ClinicChoicePage.unit.spec.jsx

```diff
@@ -148,9 +148,7 @@ describe('VAOS vaccine flow <ClinicChoicePage>', () => {
       store,
     });
 
-    await screen.findByText(/Choose a clinic located at/i);
-
-    userEvent.click(screen.getByLabelText(/red team/i));
+    userEvent.click(await screen.findByLabelText(/red team/i));
```

Comment:
I guess this is why the test is flaky. Just curious. What's the difference?

Reply:
The "Choose a clinic located at" text shows up on the page immediately, but the "red team" text, which is a clinic name, shows up after the clinics are finished being fetched and the form is rendered, which is not necessarily immediate. So this makes sure to wait for the clinic name text before clicking on it.